feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -381,21 +383,24 @@ func (a *testArgs) validateTrafficFlows(t *testing.T, flow string, expected_outg
 	otgutils.LogPortMetrics(t, a.ate.OTG(), a.top)
 	// Get send traffic
 	incoming_traffic_counters := gnmi.OC().Interface(a.ate.Port(t, "port1").Name()).Counters()
-	sentPkts := gnmi.Get(t, a.ate, incoming_traffic_counters.OutPkts().State())
+	sentPkts := gnmi.Get(t, a.ate.OTG(), incoming_traffic_counters.OutPkts().State())
```

Comment:
Needs to use port ID instead of name here and then a slightly different path here (as well for the out packet counts below):
```
incoming_traffic_state := gnmi.OTG().Port(a.ate.Port(t, "port1").ID()).State()
sentPkts := gnmi.Get(t, a.ate.OTG(), incoming_traffic_state).GetCounters().GetOutFrames()
```

Reply:
Corrected